cmd/notary-server/main.go

```diff
@@ -111,6 +113,20 @@ func main() {
 			viper.GetString("trust_service.port"),
 			viper.GetString("trust_service.tls_ca_file"),
 		)
+		minute := 1 * time.Minute
+		health.RegisterPeriodicFunc(
+			"Trust operational",
+			// If the trust service fails, the server is degraded but not
+			// exactly unheatlthy, so always return healthy and just log an
+			// error.
+			func() error {
+				err := trust.(*signer.NotarySigner).CheckHealth(minute)
```

Comment:
Sorry, going outside the scope of your commits!

If above, rather than having `trust = signer.NewNotarySigner...` we do:
``` 
notarySigner := signer.NewNotarySigner(...)
trust = notarySigner
```
we'll eventually be able to un-export NotarySigner and won't need to do the cast here.